lib/js/components/TagList/TagList.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+	<div
+		:class="{
+			tagList: true,
+			'-neutral': color === TAG_LIST_COLORS.NEUTRAL,
+			'-neutralHeavy': color === TAG_LIST_COLORS.NEUTRAL_HEAVY,
+			'-xSmall': size === TAG_LIST_SIZES.X_SMALL,
+			'-small': size === TAG_LIST_SIZES.SMALL,
+			'-medium': size === TAG_LIST_SIZES.MEDIUM,
+		}"
+	>
+		<icon class="tagList__icon" :icon="ICONS.FA_TAGS" :size="ICON_SIZES.X_SMALL" />
+		<div>{{ tagsConcatenated }}</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+.tagList {
+	$root: &;
+
+	align-items: flex-start;
+	display: flex;
+
+	&.-neutral {
+		color: $color-neutral-text-weak;
+
+		#{$root}__icon {
+			color: $color-neutral-icon-weak;
+		}
+	}
+
+	&.-neutralHeavy {
+		color: $color-neutral-text-heavy;
+
+		#{$root}__icon {
+			color: $color-neutral-icon;
```

Comment:
do we want to add transition on color to make the change smooth?

Reply:
I didn't hear anything about changing it on the fly.